Controller/SystemInfoController.php

```diff
@@ -56,4 +63,31 @@ public function phpinfoAction()
 
         return $response;
     }
+
+    /**
+     * Temporary function for getting composer package info into templates.
```

Comment:
"fixed", as method is private I don't really need to mention this, we can change it at any point.